### DISCUSSION TITLE
Suggestion: Timestamp

### DISCUSSION THREAD

**Discussion by tzvr** [maximmax42/Discord-CustomRP#96] [Ideas]
New timestamp option to make it so it's possible to display "for (time)" and not only "(time) elapsed" where it's counting just minutes and hours instead of every second. Like some regular apps. Example:

![asdf](https://user-images.githubusercontent.com/97739870/149518403-bf8ba469-90af-494d-9bfd-3c24ecd581f0.png)

**Comment by maximmax42** [comment]
I think it's showing this when you launch a known to Discord game that doesn't have rich presence. So, probably impossible for me to do, but I'll look into it.
